README.rst

```diff
@@ -69,19 +68,29 @@ Example output:
   Results:
   m=11000111111011001000
 
+Feature requests / Bugs / Questions
+--------------------------------
 
-Contributing
-------------
-
-We welcome contributions. Please follow these
-`guidelines <https://github.com/quantumlib/cirq/blob/master/CONTRIBUTING.md>`__.
+If you have feature requests or you found a bug, please `file them on Github <https://github.com/quantumlib/Cirq/issues/new/choose>`__.
 
-We use
-`Github issues <https://github.com/quantumlib/Cirq/issues>`__
-for tracking requests and bugs. Please post questions to the
+For questions about how to use Cirq post to
 `Quantum Computing Stack Exchange <https://quantumcomputing.stackexchange.com/>`__ with the
 `cirq <https://quantumcomputing.stackexchange.com/questions/tagged/cirq>`__ tag.
-For informal discussions about Cirq, join our `cirqdev <https://gitter.im/cirqdev>`__ Gitter channel.
+
+Cirq Contributors Community
+------------------
+
+We welcome contributions! Before opening your first PR, a good place to start is to read our
+`guidelines <https://github.com/quantumlib/cirq/blob/master/CONTRIBUTING.md>`__.
+
+We are dedicated to cultivate an open and inclusive community to build software for near term quantum computers.
```

Comment:
nit: "dedicated to" expects a noun expression, so use the gerund:

"We are dedicated to *cultivating* an open..."

Reply:
Thank you!!! That's my Hunglish :D